src/EnergyPlus/CMakeLists.txt

```diff
@@ -995,6 +995,7 @@ if(BUILD_PACKAGE)
       # on Windows, with Build Package, and also Link With Python, we can also drop in shortcuts to the new auxiliary CLI
       # NOTE: The install command COMPONENTS should line up so that they are run at the same packaging step
       install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE:energyplus>\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk\" -Arguments \"auxiliary eplaunch\")" COMPONENT Auxiliary)
+      install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE:energyplus>\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/IDFVersionUpdater.lnk\" -Arguments \"auxiliary updater\")" COMPONENT Auxiliary)
       install(FILES $<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk DESTINATION "./" COMPONENT Auxiliary)
```

Comment:
The IDFVersionUpdater shortcut is not there. Does it need a line like this one?

Reply:
Oh sheesh.  Yes.  I've got another round of tweaks on the updater tool itself that I'm wrapping up and will tag/pin a new version, and update this here.